api/consent-openapi.yaml

```diff
@@ -129,62 +127,50 @@ paths:
             type: string
 
         - in: query
-          name: "offset"
-          description: "Requested index for start of resources to be provided in response requested by client"
-          required: false
-          schema:
-            type: integer
-
-        - in: query
-          name: "limit"
-          description: "Requested number of resources to be provided in response requested by client"
-          required: false
+          name: policy
+          description: "An object of type Policy"
+          required: true
           schema:
-            type: integer
+            $ref: '#/components/schemas/Policy'
 
-      x-specification-usecase: "UC-C-PIC-A-003"
+      x-specification-usecase: "UC-C-PIC-A-002"
       x-specification-scenario: ""
       x-specification-pii-or-sensitive: "False"
-      x-specification-crudl-model: "Revision"
+      x-specification-crudl-model: "Policy"
       responses:
         '200':
           description: ""
             
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Policy'
+                type: array
+                items:
+                  oneOf:
+                    - $ref: '#/components/schemas/Policy'
+                    - $ref: '#/components/schemas/Revision'
 
         '400':
           description: bad input parameter
       security:
         - OAuth2: [org]
 
 
-  /config/policy/{id}/:
-
-    put:
+    delete:
       tags:
         - config
-      summary: "UPDATE - Updates an existing Policy object, returning the updated version and a new revision. Updating a Policy does not affect existing references in Agreement, the new revision should be specified for Agreement."
-      operationId: "orgUpdatePolicy"
-      description: "UPDATE - Updates an existing Policy object, returning the updated version and a new revision. Updating a Policy does not affect existing references in Agreement, the new revision should be specified for Agreement."
+      summary: "DELETE - Deletes an existing Policy object, returning the updated version and a new revision. Deleting a Policy is not possible if it's associated with active Agreement."
+      operationId: "configDeletePolicy"
+      description: "DELETE - Deletes an existing Policy object, returning the updated version and a new revision. Deleting a Policy is not possible if it's associated with active Agreement."
       parameters: 
         - in: path
-          name: "id"
+          name: "policyId"
```

Comment:
@aaviksoo I'm not going to ask for review on this PR since it only deals with consistency updates to things that were obviously wrong in the API spec.

In the above line that I've commented on, you can see the type of update that it reflects. Before we were saying just `id` as a URL parameter, which raised the question: What ID? The ID of the Policy or the ID of the Revision?

The diff is hard to read and makes matters more confusing than they're supposed to.